app/assets/stylesheets/responsive/_global_style.scss

```diff
@@ -257,9 +257,12 @@ div.pagination {
   &:active,
   &:focus {
     background-color: white;
-    border: 1px solid #ddd;
+    border: 1px solid #bbb;
```

Comment:
I'd say this could be even more obvious:

**Actions focused**

![screen shot 2016-08-31 at 16 21 10](https://cloud.githubusercontent.com/assets/282788/18135690/a231fd64-6f9a-11e6-8f72-243652a49ea0.png)

**Follow focused**

![screen shot 2016-08-31 at 16 21 14](https://cloud.githubusercontent.com/assets/282788/18135710/b25d8b18-6f9a-11e6-918e-5c065545fade.png)